ext/json/json.c

```diff
@@ -95,6 +96,9 @@ static PHP_MINIT_FUNCTION(json)
 	INIT_CLASS_ENTRY(ce, "JsonSerializable", json_serializable_interface);
 	php_json_serializable_ce = zend_register_internal_interface(&ce);
 
+	INIT_CLASS_ENTRY(ce, "JsonException", NULL);
+	php_json_exception_ce = zend_register_internal_class_ex(&ce, zend_ce_exception);
```

Comment:
I'd like this to extend `RuntimeException`, but that made PHP complain about this not implementing `Throwable` for some reason and throw an `Exception` instead. :/

Reply:
Because SPL might not be loaded is a good reason. :D

Reply:
There's probably any easy workaround. I also cba.